corebehrt/main/helper/evaluate_xgboost.py

```
@@ -0,0 +1,73 @@
+    model: xgb.Booster, fi_cfg: dict, encoding_vocab: dict
+) -> pd.DataFrame:
+    """Get feature importance for a trained XGBoost model."""
+    importance_type = fi_cfg.get("importance_type", "gain")
+    importance = model.get_score(importance_type=importance_type)
+    rev_encoding_vocab = {v: k for k, v in encoding_vocab.items()}
+    importance_df = pd.DataFrame(
+        {
+            "feature": list(importance.keys()),
+            "concept": [
+                rev_encoding_vocab[int(feature.lstrip("f"))]
+                for feature in importance.keys()
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)